controllers/networking-calico/charts/internal/calico/templates/calico.yaml

```diff
@@ -184,17 +184,12 @@ spec:
             - name: CALICO_IPV4POOL_CIDR
               value: "{{ .Values.global.podCIDR }}"
             # Enable IPIP
-            {{- if ne .Values.config.backend "none"}}
-            # Enable IP-in-IP within Felix.
-            - name: FELIX_IPINIPENABLED
```

Comment:
Yes, because of exactly that. It is automatically set to `true` when an IPIP-enabled pool is created. Do you have a case where where we want to set an IPIP pool to disabled while ` FELIX_IPINIPENABLED` is true or the other way around?

Reply:
Very well